src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.ArrayList;
+
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static void printIntro(){
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void main(String[] args) throws UnknownCommandException,EmptyInputException,java.io.IOException{
```

Comment:
It is a bit strange that your main is allowed to throw errors.
Shouldn't all errors be caught and handled by main ?